contributors/design-proposals/storage/csi-migration.md

```diff
@@ -88,16 +104,31 @@ GA [TBD]
 * GCE PD feature parity in CSI with in-tree implementation
 * AWS EBS feature parity in CSI with in-tree implementation
 * Cloud Driver feature parity in CSI with in-tree implementation
+* Skip probing/registration of migrated in-tree plugin code
 
 ## Dependency Graph
 
 ![CSI Migration Dependency Diagram](csi-migration_dependencies.png?raw=true "CSI Migration Dependency Diagram")
 
 ## Feature Gating
 
-We will have an alpha feature gate for the whole feature that can turn the CSI
-migration on or off, when off all code paths should revert/stay with the in-tree
-plugins. We will also have individual flags for each driver so that admins can
+We will have two feature gates for the overall feature: CSIMigration and
+CSIMigrationInTreeOff.
```

Comment:
`CSIMigrationCommitted`? `CSIMigrationNoFallback`? Naming is hard :)

/cc @msau42

Reply:
My main gripe with it currently is that there is a "negative" in the name. Meaning we will end up saying "lets turn *on* the in tree *off* flag" which I forsee will be confusing

Reply:
Completely agree. I initially wrote it up with `CSIMigrationInTreeDisabled` and soon realized "enabling CSIMigrationInTreeDisabled` is counter intuitive. I was also thinking of something very generic like `CSIMigrationPhase2` to avoid the "negative" enablement phrasing  but that does not convey much around what the flag actually does.